src/main/java/org/gelecekbilimde/scienceplatform/post/service/CategoryService.java

```diff
@@ -4,26 +4,30 @@
 import org.gelecekbilimde.scienceplatform.post.exception.CategoryNotFoundException;
 import org.gelecekbilimde.scienceplatform.post.model.Category;
 import org.gelecekbilimde.scienceplatform.post.model.request.CategoryCreateRequest;
+import org.gelecekbilimde.scienceplatform.post.model.request.CategoryUpdateRequest;
 
 import java.util.List;
 
 public interface CategoryService {
 
-	List<Category> getCategories();
-
 	Category getCategory(Long categoryId);
 
-	/**
-	 * Creates a new category.
-	 * If there is a category with same order, it increases the order of the siblings (categories with the same parent).
-	 *
-	 * @param request The request object that contains the name,order and if it has a parent, parentId of the category.
-	 * @throws CategoryAlreadyExistException      throws if a category with the same name already exists.
-	 * @throws CategoryNotFoundException throws if there is no category with the given parentId.
-	 */
+	List<Category> getCategories();
```

Comment:
Bu metodun en üstte olması daha iyi olabilir sıralama olarak;

1. Listeleme
2. Listeleme Özet (Summary)
3. Detay
4. Oluşturma
5. Güncelleme
6. Silme
7. Diğer aksiyonlar

```suggestion
	List<Category> findAll();
```